src/app/savings/savings-account-view/transactions/view-transaction/view-transaction.component.scss

```
@@ -35,11 +18,53 @@ mifosx-account-header {
 }
+
+.customer-title,
+.customer-metadata,
+.customer-summary-column,
+.customer-summary td,
+mifosx-entity-name,
+mifosx-account-number,
+mifosx-external-identifier {
+  min-width: 0;
+  overflow-wrap: anywhere;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)